cli/scaletest.go

```diff
@@ -902,10 +902,10 @@ func (r *RootCmd) scaletestWorkspaceTraffic() *clibase.Cmd {
 				_, _ = fmt.Fprintln(inv.Stderr, "\nUploading traces...")
 				if err := closeTracing(ctx); err != nil {
 					_, _ = fmt.Fprintf(inv.Stderr, "\nError uploading traces: %+v\n", err)
-					// Wait for prometheus metrics to be scraped
-					_, _ = fmt.Fprintf(inv.Stderr, "Waiting %s for prometheus metrics to be scraped\n", scaletestPrometheusWait)
-					<-time.After(scaletestPrometheusWait)
 				}
+				// Wait for prometheus metrics to be scraped
```

Comment:
nit: I'm wondering if there could be an active loop implemented instead to make sure that metrics have been scraped. Nothing is wrong with this submission though!

Reply:
I considered that as well but didn't think it worth the additional complexity.